### ISSUE TITLE
Add warning to the "Drive" feature, reminding end users it is NOT secure/private storage 

### ISSUE THREAD

**Issue by dalekcoffee** [syuilo/Misskey#13651]
### Summary

When using with S3 storage set to public, I think it would be wise to be able to place a warning/disclaimer in the Drive feature so end users understand the risk of uploading any media/file in there that is sensitive in nature.

Preferably something that the end user has to dismiss for acknowledgement, or is always present but not too invasive/obstructive. Or a combination of both perhaps

### Purpose

Increase transparency, protect the end users from themselves

### Do you want to implement this feature yourself?

- [ ] Yes, I will implement this by myself and send a pull request

**Comment by CupofDalek** [comment]
> (actually i saw your merge request in sharkey repository, and i ported back to the misskey hub, where the IMPORTANT_NOTES.md originally belongs to)
> 
> [misskey-dev/misskey-hub-next#130](https://github.com/misskey-dev/misskey-hub-next/pull/130)

Yeah I guess I was trying to advocate for it being present IN the UI even from misskey core perhaps as opposed to those notes only.

My mindset being that while I WANT my end users to read all the info, privacy policy, TOS. I just know not all users do. and yes it creates a scenario where failure to understand that is kind of "their fault" but my heart is in the right place such as trying to make sure they are protected and even their own ignorance cant avoid a visible warning on the drive page itself lol. 

I REALLY don't trust the common end user haha, and I guess I wanted to see if the misskey core community would share some of that sentiment and would also have an interest in adding a layer of awareness, or if they would also stand firm against that mindset, and leave it up to the end user
